include/HUDL/HUDL.hpp

```diff
@@ -30,37 +44,237 @@ class HUDL {
+         * 1: Vendor ID
+         * 2: Product Code
+         * 3: Revision Number
+         * 4: Serial Number
+         */
+        {
+            .Key = CO_KEY(0x1018, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x10,
+        },
+        {
+            .Key = CO_KEY(0x1018, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x11,
+        },
+        {
+            .Key = CO_KEY(0x1018, 3, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x12,
+        },
+        {
+            .Key = CO_KEY(0x1018, 4, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x13,
+        },
+
+        /**
+         * SDO CAN message IDS.
+         * 1: Client -> Server ID, default is 0x600 + NODE_ID
+         * 2: Server -> Client ID, default is 0x580 + NODE_ID
+         */
+        {
+            .Key = CO_KEY(0x1200, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x600 + NODE_ID,
+        },
+        {
+            .Key = CO_KEY(0x1200, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x580 + NODE_ID,
+        },
+
+        /**
+         * RPDO0 settings
+         * 0: RPDO number in index and total number of sub indexes.
+         * 1: The COB-ID to receive PDOs from.
+         * 2: transmission trigger
+         */
+        {
+            .Key = CO_KEY(0x1400, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 3,
+        },
+        {
+            .Key = CO_KEY(0x1400, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = TMS_TPDO0_COB_ID,
+        },
+        {
+            .Key = CO_KEY(0x1400, 2, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0xFE,
+        },
+
+        /**
+         * RPDO1 settings
+         * 0: RPDO number in index and total number of sub indexes.
+         * 1: The COB-ID to receive PDOs from.
+         * 2: transmission trigger
+         */
+        {
+            .Key = CO_KEY(0x1401, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 3,
+        },
+        {
+            .Key = CO_KEY(0x1401, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = TMS_TPDO1_COB_ID,
+        },
+        {
+            .Key = CO_KEY(0x1401, 2, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0xFE,
+        },
+
+        /**
+         * RPDO0 mapping, determines the PDO messages to send when RPDO0 is triggered
+         * 0: The number of PDO message associated with the RPDO
+         * 1: Link to the first PDO message - tempThree
+         * 2: Link to the second PDO message - tempFour
+         */
```

Comment:
Your comment appears to be a duplicate of RPDO map 1 and does not match the variables linked to in this map.